src/Microsoft.AspNetCore.Server.IntegrationTesting.IIS/IISDeployer.cs

```diff
@@ -22,8 +22,17 @@ public partial class IISDeployer : ApplicationDeployer
         public IISDeployer(DeploymentParameters deploymentParameters, ILoggerFactory loggerFactory)
             : base(deploymentParameters, loggerFactory)
         {
+            IISDeploymentParameters = (IISDeploymentParameters)deploymentParameters;
         }
 
+        public IISDeployer(IISDeploymentParameters deploymentParameters, ILoggerFactory loggerFactory)
+            : base(deploymentParameters, loggerFactory)
+        {
+            IISDeploymentParameters = deploymentParameters;
+        }
+
+        public IISDeploymentParameters IISDeploymentParameters { get; }
```

Comment:
I would like to have IISDeployer and IISExpressDeployer have a shared base class, however I would prefer to have it in a separate PR.